djangoplicity/media/tasks.py

```diff
@@ -600,7 +600,7 @@ def upload_youtube(video_id, user_id=None):
 
     def mail_user(subject, body=''):
         if user and user.email:
-            send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, [user.email])
+            send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, settings.DEFAULT_MAIL_USER)
```

Comment:
This is not recommended because you are adding settings that are not already in Djangoplicity nor in Django, this would break if they are not set. I recommend using a different approach.

Reply:
Also, this is intended to inform the user that produced the task, and this change would remove that possibility.

Reply:
Good afternoon Edison, they assigned me the task of using 2 variables that were already added in the Hubble repository, the idea is that now it is sent to a hubble mail and thus a ticket arrives to mantis, in the second case they asked me to create one variable specific to him.

DEFAULT_MAIL_USER = 'web@esahubble.org'
DEFAULT_MAIL_TAGGING = 'zidmani@gmail.com'

This configuration was requested by Engineer Javier

Reply:
Well, it's ok but the idea would be to keep all djangoplicity projects aligned, this change it's only for bubble which is ok, but this could create a hard fork between hubble and other projects. That's why I recommend just checking for the existence of the setting and providing a default value, like this:
```suggestion
            send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, getattr(settings, 'DEFAULT_MAIL_USER', user.email)
```

Reply:
You are absolutely right, thank you for your contribution, it is very useful.